src/inputs/NumberField.tsx

```diff
@@ -42,6 +45,7 @@ export function NumberField(props: NumberFieldProps) {
   } = props;
 
   const factor = type === "percent" || type === "cents" ? 100 : type === "basisPoints" ? 10_000 : 1;
+  const labelSuffix = getLabelSuffix(required);
```

Comment:
**Question:** Wondering why we are not utilizing the `formState` object to determine `required` (using the Bound fields)? I'm concerned that we could have `formState` with required but not set that on the `FormLine` - maybe I'm not following.

Reply:
It does happen that way:

* A page has `const config = { firstName ... rules: [required] }`
* The page creates a `formState`
* The page does `BoundTextField field=formState.firstName`
* `BoundTextField` sets `TextField.required` based on `firstName.required`

Reply:
(So, more concretely, this `NumberField` component itself isn't using `formState` b/c our `src/inputs` fields are decoupled from `formState`, and it's the `Bound...Field`s that are the `formState` <-> `src/inputs` glue. So that mapping of "take the form fields `required`-ness, based on the rule", happens in the `BoundNumberField`.)